pages/test_analytics/ci_environments.md

```diff
@@ -4,14 +4,25 @@ Buildkite Test Analytics collectors automatically detect common continuous integ
 If available, test collectors gather information about your test runs, such as branch names and build IDs.
 Test collectors gather information from the following CI environments:
 
-- Buildkite
-- CircleCI
-- GitHub Actions
+- [Buildkite](/docs/test-analytics/ci-environments#buildkite)
+- [CircleCI](/docs/test-analytics/ci-environments#circleci)
+- [GitHub Actions](/docs/test-analytics/ci-environments#github-actions)
 
-If you run test collectors inside containers or you're using another CI system, then you must set variables to report your CI details to Buildkite.
+If you run test collectors inside [containers](/docs/test-analytics/ci-environments#containers-and-test-collectors) or you're using another CI system, then you must set variables to report your CI details to Buildkite.
 
 If you're not using a test collector, see [Importing JSON](/docs/test-analytics/importing-json) and [Importing JUnit XML](/docs/test-analytics/importing-junit-xml) to learn how to provide run environment data.
 
+## Recommended environment variables
+If you're manually providing Buildkite Test Analytics environment variables, we'd strongly recommend setting the following variables:
```

Comment:
A couple of style thoughts:

- Let's drop the _'d_ to make this a bit simpler and stronger.
- I don't think we need to include the product name.

```suggestion
If you're manually providing environment variables, we strongly recommend setting the following variables:
```